autoscheduler/scraper/serializers.py

```diff
@@ -36,3 +37,56 @@ def get_meetings(self, obj): # pylint: disable=no-self-use
             'end_time': meeting.end_time,
             'type': meeting.meeting_type,
         } for meeting in meetings]
+
+class TermSerializer(serializers.ModelSerializer):
+    """ Serializes a term into an object with information needed by /api/terms """
+    desc = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Department
+        fields = ['term', 'desc']
+
+
+    def get_desc(self, obj):
+        """ Uses term field to generate description for the term in the
+            form "Fall - College Station" format"""
```

Comment:
```suggestion
        """ Uses term field to generate description for the term in the
            format "Fall - College Station"
        """
```
All multi-line docstrings should be formatted like this (new line for closing quotation marks), there are a couple more that you need to change the formatting the same way. Also notice the change to the comment text